System/Console/ANSI/Color.hs

```diff
@@ -26,15 +31,18 @@ import System.Console.ANSI.Types
 
 -- | Returns a color, in the sRGB colorspace, whose components are the
 -- components of an 8-bit ANSI color.
```

Comment:
I do not think the 'ANSI' standard specifies that the paletted colour should follow the xterm 256 colour palette protocol. On that basis, perhaps something like this would be a better description: 'Returns the color in the sRGB colorspace corresponding to the color in the xterm 256-color palette.'